book_maker/loader/epub_loader.py

```
@@ -458,14 +578,285 @@ def process_item(
+                )
+
+        chapter_helper = ChapterHelper(
+            self, translator, chapter_context_list, chapter_translated_list
+        )
+
+        for i in range(len(p_list)):
+            p = p_list[i]
+            temp_p = copy(p)
+
+            for p_exclude in self.exclude_translate_tags.split(","):
+                if type(p) == NavigableString:
```

Ruff (0.14.10)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)